lib/checks/navigation/region.js

```diff
@@ -17,18 +17,30 @@ const implicitLandmarks = landmarkRoles
 	.filter(r => r !== null);
 
 // Check if the current element is the skiplink
-function isSkipLink (node) {
-	return skipLink && skipLink === node;
+function isSkipLink (vNode) {
+	return skipLink && skipLink === vNode.actualNode;
 }
 
 // Check if the current element is a landmark
-function isLandmark (node) {
-	if (node.hasAttribute('role')) {
-		return landmarkRoles.includes(node.getAttribute('role').toLowerCase());
+function isLandmark (virtualNode) {
+	var node = virtualNode.actualNode;
+	var explictRole = (node.getAttribute('role') || '').trim().toLowerCase();
+
+	if (explictRole) {
+		if (explictRole === 'form') {
+			return !!aria.labelVirtual(virtualNode);
+		} else {
```

Comment:
Small point, but I'd like to see these "else" blocks removed where they simply return the "happy path":
```js
if (explicitRole === 'form') {
  // edge case
  return !!aria.labelVirtual(virtualNode);
}
// common case
return landmarkRoles.includes(explicitRole);
```